### ISSUE TITLE
Anzeige der Auflagen als Ordnungskriterium für die Abzüge (interner Bereich)

### ISSUE THREAD

**Issue by cnoss** [lucascranach/cda-orga#114]

**Comment by yannic-bruegger** [comment]
Ich nehme mir mal dieses Issue und bereite es frontend-seitig vor. 👍

**Comment by yannic-bruegger** [comment]
Habe mal ein wenig vorbereitet. Die Auflagen sind jetzt drin. Leider kann ich aktuell die Auflagen noch nicht einem Zustand zuordnen. Woher weiß ich, zu welchem Zustand welche Auflagen gehören. Aktuell fehlt mir außerdem noch die Info, zu welcher Auflage ein Druck gehört (aber ich denke das kommt dann bald durch den Importer?).

![Image](https://github.com/lucascranach/cda-orga/assets/36576062/146245d0-1d17-46e6-9fd7-dea73ce2b612)

**Comment by dangoerr** [comment]
Im virtuellen Datensatz sind nach bisherigem Plan nur die Auflagen (im Feld Datierung) aufgelistet, Angaben zu den Zusänden gibt es dort nicht, die kommen also erst im unteren Bereich der Seite zum Einsatz. Jedem Exemplardatensatz ist ein Zustands- und eine Auflagenangabe zugeordnet aus der sich die Reihenfolge ableitet. Beantwortet das die Frage?

**Comment by yannic-bruegger** [comment]
Das hilft mir schon, danke!

In der bisherigen Implementation gehen wir von maximal fünf Zusänden aus. Ich mache mir noch sorgen, dass diese Implementation und Annahme ein paar Dinge schwieriger macht. Besonders auf längere Zeit betrachtet.

```js
const reprintsLevel1 = getReprints(this, data, 1, 0);
const reprintsLevel2 = getReprints(this, data, 2);
const reprintsLevel3 = getReprints(this, data, 3);
const reprintsLevel4 = getReprints(this, data, 4);
const reprintsLevel5 = getReprints(this, data, 5);
```
_aus graphic-virtual-object.11ty.js_

Ich nutze jetzt alle Datierungen die als `EDITION` (Auflage?) im virtuellen Datensatz hinterlegt sind um die ein- und ausklappbaren Bereiche inklusive Beschreibungstext anzuzeigen. 

Am leichtesten für die technische implementation wäre folgende Struktur:
Liste aller Zustände -> Liste aller Auflagen pro Zustand -> Liste aller Reprints pro Auflage

So wie es jetzt ist bzw. bald sein soll funktioniuert aber auch. Dann wird die implementation nur etwas wilder 😄

**Comment by dangoerr** [comment]
Ja, EDITION scheint das korrekt Feld zu sein. Ich fürchte, wir werden die wildere Implementation nutzen müssen, denn die Listen aller Zustände, Auflagen und Reprints werden erst am Ende des Projektes vollständig sein. Wir haben zwar bereits ein grobes Gerüst, aber mit den wachsenden Erkenntnissen während des Projekts werden sich diese Listen weiter ausdifferenzieren.

**Comment by yannic-bruegger** [comment]
``json
md: {
  'Unbekannter Zustand': {
		'Auflage a)': 
			Reprints...
		'Auflage b)': 
			Reprints...
		'Auflage c)': 
			Reprints...
	}
	'1. Zustand': {
		'Auflage d)': 
			Reprints...
		'Auflage e)': 
			Reprints...
		'Auflage f)': 
			Reprints...
	}
	'2. Zustand': {
		'Auflage g)': 
			Reprints...
		'Auflage h)': 
			Reprints...
	}
}
``

**Comment by yannic-bruegger** [comment]
Wie ist hier der Stand? Status ist auf Testing/Waiting - ich bin mir aber nicht sicher auf wen. Ich stelle es mal auf in progress bei Markus. Das ist der letzte Stand, dne ich vor seinem Urlaub hatte. Ansonsten mich gerne korrigieren :)

**Comment by markusalterauge** [comment]
Kurze Nachfrage: Ich meine wir hatten gesagt, dass wir die Infos in historicEventInformations[] ablegen wollten. Wenn ich das richtig sehe ist für cranach-artefacts eher die references[].reprints[] interessant. Würde eher dort die Struktur implementieren.

**Comment by markusalterauge** [comment]
@dangoerr ist es möglich, in den historischen Daten nicht nur die Auflage, sondern auch den Zustand zu hinterlegen?

**Comment by dangoerr** [comment]
Leider nein, da wir knapp 1000 schon vorhandene Datensätze händisch ändern müssten.

**Comment by dangoerr** [comment]
Feedback zur aktuellen Stand-alone-Lösung (Danke dafür!) zur Darstellung der Auflagen (wie mit Markus besprochen, es geht dann doch recht schnell):
- Grundstruktur und dargestellte Werke alles Bestens
- Es sollten jedoch default neben den Auflagen-Buchstaben (a, b, c...) und ihrer Datierung auch die Thumbs der Werke/Abzüge sichtbar sein.
- D.h. default eingeklappt bitte nur der Text, der die jeweilige Auflage beschreibt
- Die Pfeile zum Ein-/Aufklappen bitte noch an das Design der Gesamtseite anpassen (hättet ihr wahrscheinlich ohnehin noch gemacht)
- Hier die Zielvorstellung zur Defaultansicht und der Ansicht mit ausgeklapptem Text visualisiert:



![Image](https://github.com/lucascranach/cda-orga/assets/91158095/7597b04b-993c-47a9-92df-662efe381be9)








![Image](https://github.com/lucascranach/cda-orga/assets/91158095/2cd625d9-5961-442f-b26a-d55181f6b898)

**Comment by yannic-bruegger** [comment]
@yannic-bruegger nimmt es in die nächste woche mit zur implementation

**Comment by yannic-bruegger** [comment]
Implementation ist fertig :)

Default:


![Image](https://github.com/lucascranach/cda-orga/assets/36576062/93cd1c37-22ba-49bd-b52c-4162c745bafa)


Expanded:


![Image](https://github.com/lucascranach/cda-orga/assets/36576062/7d29b5a5-e1de-44f3-9739-b8ac9289dcba)

**Comment by yannic-bruegger** [comment]
Ich würde euch die Version zum testen bereitstellen, nachdem ihr #71 getestet habt. Dann kann das live gehen und wir überschreiben nichts :) LG

**Comment by dangoerr** [comment]
Hey, sorry, war länger unterwegs. Du kannst das in den internen Bereich gerne jederzeit aufnehmen.

**Comment by yannic-bruegger** [comment]
Deployment-Strategie wird morgen besprochen

**Comment by yannic-bruegger** [comment]
https://lucascranach.org/intern/artefacts-preview/de/LC_HVI-56_79/

**Comment by yannic-bruegger** [comment]
Es gibt noch Punkte, die wir diskutieren müssen:

Ist es eine Ausgabe oder eine Auflage? In der Übersetzungstabelle von uns steht folgendes:

"edition": {
    "de": "Ausgabe",
    "en": "Edition"
  },

Ich kann allerdings nicht ohne weiteres sagen, ob diese Übersetzung aus einem anderen Kontext (z.B. Literatur?) kommt. Ggf. müssen wir das dann in den Übersetzungen trennen, auch wenn es auf Englisch identisch ist.

**Comment by cnoss** [comment]
@cnoss im internen Bereich KKL prüfen

**Comment by cdaJana** [comment]
hier zwei Bsps. für die fehlenden Abzüge des KKL im internen Bereich![Image](https://github.com/user-attachments/assets/e6fccbdd-0096-48ca-b7fe-2547345e74e0)
![Image](https://github.com/user-attachments/assets/2fbdefc5-a021-49c5-b31d-33415370c2a2)

**Comment by dangoerr** [comment]
ACHTUNG! Aktualisierung: Weiterhin gilt, dass die Auflagen als Ordnungskriterium der Abzüge benötigt werden (siehe pptx oben, Folie 1 und 2).
NICHT mehr, wird eine Darstellung der Auflagencharakteristika in Textform bei den Abzügen benötigt. Folie 3 der pptx entfällt also.

Da reicht völlig, wie es schon jetzt bei LC_HVI-56_79 im internen Bereich umgesetzt ist:
![Image](https://github.com/user-attachments/assets/816169f2-6ddb-4300-9410-edae9bfefa70)

**Comment by vschaefer** [comment]
@cnoss Hast du hier zu schon Feedback bekommen?
